### PR TITLE
[D1] add a page for remote development

### PR DESCRIPTION
**Pull request by rozenmd**
I wasn't actually sure where this page should live, but the content should definitely be in our docs.

### CODE REVIEW

**Review by elithrar — APPROVED** [review]

File: content/d1/learning/remote-development.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Remote development
+weight: 6
+pcx_content_type: concept
+---
+
+# Develop remotely
+
+D1 has support for remote development, using the [dashboard playground](/workers/learning/playground/#using-the-playground). The dashboard playground uses a browser version of VS Code, letting you rapidly iterate on your Worker entirely in your browser.
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
D1 supports remote development using the [dashboard playground](/workers/learning/playground/#using-the-playground). The dashboard playground uses a browser version of Visual Studio Code, allowing you to rapidly iterate on your Worker entirely in your browser.
```

File: content/d1/learning/remote-development.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Users new to D1 and/or Cloudflare Workers should visit the [D1 tutorial](/d1/get-started/) to install `wrangler` and deploy their first database.
+
+{{</Aside>}}
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
+2. Go to [**Workers & Pages** > **Overview**](https://dash.cloudflare.com/?to=/:account/workers-and-pages).
+3. Select an existing Worker.
+4. Select the **Settings** tab.
+5. Select the **Variables** sub-tab.
+6. Scroll down to the **D1 Database Bindings** heading
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
6. Scroll down to the **D1 Database Bindings** heading.
```

File: content/d1/learning/remote-development.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+7. Enter a variable name, such as `DB`, and pick the D1 database you wish to access from this Worker
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
7. Enter a variable name, such as `DB`, and select the D1 database you wish to access from this Worker.
```

File: content/d1/learning/remote-development.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+8. Click **Save and deploy**
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
8. Select **Save and deploy**.
```

File: content/d1/learning/remote-development.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+title: Remote development
+weight: 6
+pcx_content_type: concept
+---
+
+# Develop remotely
+
+D1 has support for remote development, using the [dashboard playground](/workers/learning/playground/#using-the-playground). The dashboard playground uses a browser version of VS Code, letting you rapidly iterate on your Worker entirely in your browser.
+
+## Bind a D1 database to a Worker
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
## 1. Bind a D1 database to a Worker
```

File: content/d1/learning/remote-development.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
+2. Go to [**Workers & Pages** > **Overview**](https://dash.cloudflare.com/?to=/:account/workers-and-pages).
+3. Select an existing Worker.
+4. Select the **Settings** tab.
+5. Select the **Variables** sub-tab.
+6. Scroll down to the **D1 Database Bindings** heading
+7. Enter a variable name, such as `DB`, and pick the D1 database you wish to access from this Worker
+8. Click **Save and deploy**
+
+## Start a remote development session
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
## 2. Start a remote development session
```

File: content/d1/learning/remote-development.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+1. Scroll to the top of the page and click **Quick edit**
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
1. On the top of the page, select **Quick edit**.
```

File: content/d1/learning/remote-development.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+2. Your Worker now has access to D1
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
2. Your Worker now has access to D1.
```

File: content/d1/learning/remote-development.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+Using the following Worker script, you can verify that the Worker has access to the bound D1 database:
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
Use the following Worker script to verify that the Worker has access to the bound D1 database:
```

File: content/d1/learning/remote-development.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+D1 has support for remote development, using the [dashboard playground](/workers/learning/playground/#using-the-playground). The dashboard playground uses a browser version of VS Code, letting you rapidly iterate on your Worker entirely in your browser.
+
+## Bind a D1 database to a Worker
+
+{{<Aside type="note">}}
+
+This guide assumes you have previously created a Worker, and a D1 database.
+
+Users new to D1 and/or Cloudflare Workers should visit the [D1 tutorial](/d1/get-started/) to install `wrangler` and deploy their first database.
```
Comment (Maddy-Cloudflare) [inline content/d1/learning/remote-development.md]:
```suggestion
Users new to D1 and/or Cloudflare Workers should read the [D1 tutorial](/d1/get-started/) to install `wrangler` and deploy their first database.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Comment by Maddy-Cloudflare** [comment]
@rozenmd do you have permission to merge this?

**Comment by rozenmd** [comment]
@Maddy-Cloudflare - yep!

**Pull request merged**